config/files/usr/etc/NetworkManager/conf.d/rand_mac.conf

```diff
@@ -0,0 +1,9 @@
+[device-mac-randomization]
+# "yes" is already the default for scanning
+wifi.scan-rand-mac-address=yes
```

Comment:
if this is already default, why do we need to set it?

Reply:
i thought about that, then decided to leave it in bc i though declaring all randomization options explicitly is cleaner. Would you remove it?

Reply:
No that's fine.